### PR TITLE
[ci] Move 2 more login specs from feature to controller spec

### PR DESCRIPTION
**Pull request by coolo**
proxy mode works with http headers, that you aren't able to control
as browser. So test the controller code (slightly misusing the
session controller)

### CODE REVIEW

File: src/api/spec/controllers/webui/session_controller_spec.rb [hunk 1]
```diff
@@ -122,4 +122,43 @@
...
+      expect(User.current).to eq(user)
+    end
+
+    it 'does not log in any user when no header is set' do
+      get :new
+      expect(User.current.login).to eq '_nobody_'
+    end
+
+    it 'creates a new user account if user does not exist in OBS' do
+      # check we're fine
```
Comment (Ana06) [inline src/api/spec/controllers/webui/session_controller_spec.rb]: why? how can that not be fine?

Comment (coolo) [reply src/api/spec/controllers/webui/session_controller_spec.rb]: @Ana06 - this are machines. If you trust the machines, you're part of the problem!

**Comment by ChrisBr** [comment]
@coolo this is necessary to switch to chromedriver?

**Comment by coolo** [comment]
well, to be precise: it's necessary to switch away from poltergeist/phantomjs

File: src/api/spec/controllers/webui/session_controller_spec.rb (hunk 1 above)
Comment (hennevogel) [reply src/api/spec/controllers/webui/session_controller_spec.rb]: Funny comments are not so funny anymore when you come back to them 3 years later and have to figure out what they mean...

Comment (coolo) [reply src/api/spec/controllers/webui/session_controller_spec.rb]: Spaßbremse!

**Comment by coolo** [comment]
@Ana06 I removed the sanity check

File: src/api/spec/controllers/webui/session_controller_spec.rb (hunk 1 above)
Comment (Ana06) [reply src/api/spec/controllers/webui/session_controller_spec.rb]: well... I don't trust this comment... how could it be that we are not fine? can you please elaborate? :thinking:

**Comment by coolo** [comment]
Well, I just like to double check my assumptions and not rely on some external factors. I pulled enough hair for stupid reasons - and just look at me, there is nothing left to pull

**Pull request merged**